action.yml

```diff
@@ -2,7 +2,7 @@ name: 'Beanstalk Deploy'
 description: 'Deploy a zip file to AWS Elastic Beanstalk'
 author: 'Einar Egilsson'
 runs:
-  using: 'node16'
+  using: 'node18'
```

Comment:
Can we change this to `node20` instead as recommended here? https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20/